packages/components/src/side_scroller/index.jsx

```diff
@@ -0,0 +1,140 @@
+  const startPositionRef = useRef(null);
+  const preventClickRef = useRef(null);
+  const stopListeningToResizeRef = useRef(null);
+
+  const className = clsx(styles.root, passedClassName);
+  const cleanProps = props;
+
+  // Fixes issue with most browsers reducing size of the scrollable element children
+  // as if to compensate for scrollbars, even when they are hidden
+  const style = { height };
+
+  const getScrollableNode = () => containerRef.current;
+
+  const stopScrollAnimation = () => {
+    if (animationIdRef.current) global.cancelAnimationFrame(animationIdRef.current);
+  };
+
+  const updateScroll = () => {
+    const { scrollLeft } = getScrollableNode();
+    const maxScrollPosition = contentWidthRef.current - containerWidthRef.current;
+
+    const nextCanScrollLeft = scrollLeft > 0;
+    const nextCanScrollRight = scrollLeft < maxScrollPosition;
+
+    setCanScrollLeft(nextCanScrollLeft);
+    setCanScrollRight(nextCanScrollRight);
+  };
+
+  const updateSizes = () => {
+    const { width: containerWidth } = size(containerRef.current);
+    const { height: nextHeight } = size(contentRef.current.getNode());
+
+    containerWidthRef.current = containerWidth;
+    contentWidthRef.current = getScrollableNode().scrollWidth;
+
+    updateScroll();
+    setHeight(nextHeight);
+  };
+
+  const handleDragStart = (event) => {
+    // Prevents DOM nodes like images from being dragged
+    event.preventDefault();
+    startXRef.current = eventCoordinates(event, 'pageX').pageX;
+    startPositionRef.current = getScrollableNode().scrollLeft;
+    stopScrollAnimation();
+  };
+
+  const handleDragStop = () => { };
+
+  const handleDrag = (event) => {
+    const diff = startXRef.current - eventCoordinates(event, 'pageX').pageX;
+    const shift = Math.abs(diff);
+    const newPosition = startPositionRef.current + diff;
+
+    if (shift >= DISABLE_SCROLL_DISTANCE) {
+      preventClickRef.current = true;
+
+      // Prevents vertical scroll on touch devices
+      event.preventDefault();
+    }
+
+    getScrollableNode().scrollLeft = newPosition;
+  };
+
+  const handleClickCapture = (event) => {
+    if (preventClickRef.current) {
+      stopEvent(event);
+      preventClickRef.current = false;
+    }
+  };
+
+  const handleAnimateScroll = (callback) => {
+    animationIdRef.current = global.requestAnimationFrame(callback);
+  };
+
+  useEffect(() => {
+    stopListeningToResizeRef.current = eventproxy('resize', updateSizes);
+    updateSizes();
+
+    return () => {
+      stopListeningToResizeRef.current();
+      stopScrollAnimation();
+    };
+  }, []);
+
+  useEffect(() => {
+    updateSizes();
+  }, [children]);
+
+  return (
+    <article {...cleanProps} className={className}>
+      <div
+        className={styles.container} ref={containerRef} style={style}
+        onScroll={updateScroll} onLoad={updateSizes}
+      >
+        <Draggable
+          ref={contentRef}
+          onDragStop={handleDragStop}
```

Comment:
If you remove handleDragStop - remove this prop as well

Reply:
Relates to what I mentioned in https://github.com/goodhood-eu/goodhood/pull/121#discussion_r714605150, let's go trough this one together!